onnxruntime/core/providers/cpu/ml/tree_ensemble_common.h

```diff
@@ -726,6 +726,9 @@ TreeEnsembleCommon<InputType, ThresholdType, OutputType>::ProcessTreeNodeLeave(
       case NODE_MODE::BRANCH_NEQ:
         TREE_FIND_VALUE(!=)
         break;
+      case NODE_MODE::BRANCH_MEMBER:
```

Comment:
It seems #21222 is still needed.

Reply:
This was added for completeness and shouldn't trigger.